Shared/Samples/Animate 3D graphic/README.md

```diff
@@ -0,0 +1,60 @@
+# Animate 3D graphic
+
+An `OrbitGeoElementCameraController` follows a graphic while the graphic's position and rotation are animated.
+
+![Image of animate 3D graphic](animate-3d-graphic.png)
+
+## Use case
+
+Visualize movement through a 3D landscape.
+
+## How to use the sample
+
+Animation Controls (Top Left Corner):
```

Comment:
Should this indicate the bottom left corner instead?

Reply:
I had forgotten to remove that.